crates/pera-framework/packages/pera-system/sources/dwallet_2pc_mpc_ecdsa_k1.move

```diff
@@ -581,6 +635,7 @@ module pera_system::dwallet_2pc_mpc_ecdsa_k1 {
             });
             i = i + 1;
         };
+        vector::reverse(&mut message_approvals);
```

Comment:
Why?

Reply:
Read the function code. At each iteration we pop the last message, so the message approvals are "reversed", until this line, in respect to the messages